### ISSUE TITLE
Wrong icon printing in Windows Phone 8.1 devices

### ISSUE THREAD

**Issue by mowcixo** [driftyco/ionicons#144]
In Internet Explorer 11 for Windows Phone, the icons prints as a rectangle. Editing the **@font-face** in **ionicons/scss/_ionicons-font.scss** like this:

 ```scss
@font-face {
  font-family: $ionicons-font-family;
  src:url("#{$ionicons-font-path}/ionicons.eot");
  src:url("#{$ionicons-font-path}/ionicons.eot") format("embedded-opentype"),
   url("#{$ionicons-font-path}/ionicons.ttf") format("truetype"),
   url("#{$ionicons-font-path}/ionicons.woff") format("woff"),
   url("#{$ionicons-font-path}/ionicons.svg") format("svg");
  font-weight: normal;
  font-style: normal;
}
```

Basically removing the query string in font file paths, solves the problem. I don't know if its a right change, but I suggest it as a solution.

**Comment by oliversalzburg** [comment]
Yay for @Ionitron and closing unresolved issues :cake: :+1: :tada:

**Comment by mhartington** [comment]
We're not supporting WP8, something we've addressed in http://blog.ionic.io/announcing-ionic-1-2/

**Comment by oliversalzburg** [comment]
@mhartington Yet, you have WP8 supporting code in your code base.

Regardless, I was trying to make a point that Ionitron closes unresolved issues and I have a hard time understanding the benefits of that.

**Comment by Suven** [comment]
Although this is quite old now.. the fix might be very easy and WP8-unrelated. 

In ttf there is [a flag that states whether or not the font is embeddable](https://en.wikipedia.org/wiki/TrueType#Embedding_protection). 

For that one to be included I opened #362 some time ago.
